inst/python_cmd.m

```diff
@@ -465,9 +473,9 @@
 %! if (exist ('OCTAVE_VERSION', 'builtin'))
 %! s = ['abc'; 'defgh'; '12345'];
 %! q = python_cmd ('return len(_ins)', s);
-%! assert (q, 1)
+%! assert (q == 1)
```

Comment:
yes, but this test doesn't care about any of that, just want to make sure q has unit length.